packages/amplify_authenticator/lib/src/widgets/form_fields/sign_up_form_field.dart

```diff
@@ -739,4 +800,9 @@ class _SignUpDateFieldState extends _SignUpFormFieldState<String>
       isOptional: isOptional,
     );
   }
+
+  @override
+  Iterable<String>? get autofillHints => const [
+        AutofillHints.birthday,
+      ];
```

Comment:
I think you can remove this. Setting it in `_SignUpFormFieldState` should be enough since `_SignUpDateFieldState` extends `_SignUpFormFieldState`.